src/containers/StructurePage/folderComponents/FolderItem.jsx

```diff
@@ -94,47 +84,57 @@ const FolderItem = ({
       />
     );
 
+  const jumpToResources = isMainActive &&
+    type === 'topic' && (
+      <Button
+        outline
+        className={css`
+          margin: 3px ${spacing.xsmall} 3px auto;
+          ${fonts.sizes(14, 1.1)};
+        `}
+        type="button"
+        onClick={() => resourceSection && resourceSection.scrollIntoView()}>
+        {t('taxonomy.jumpToResources')}
+      </Button>
+    );
+
   return (
-    <React.Fragment>
-      <div id={uniqueId} data-cy="folderWrapper" {...classes('wrapper')}>
-        <FolderLink
-          toLink={toLink}
-          name={name}
-          active={active}
-          grayedOut={grayedOut}
-        />
-        {showLinkButton}
-        {editLinkButton}
-        {settingsButton}
-        {subjectFilters}
-      </div>
-      <SubFolders {...sendToSubFolders} />
-    </React.Fragment>
+    <div data-cy="folderWrapper" {...classes('wrapper')}>
+      {showLinkButton}
```

Comment:
Vet at det har vært sånn lenge, men hvorfor kan ikke dette være komponenter isteden? Nå sitter vi med en lang komponent som bare er vanskelig å lese.

Reply:
Åpen for forslag. Du vil ha egen komponent i egen fil for hver knapp?

Reply:
Ja, tror det blir lettere, evt en type knapp som kan gjenbrukes. Felles for alle knapper er jo at de har en condition så kunne jo hatt en komponent som det her:

```
<SomeTaxButton
  outline
  show={isMainActive && type === 'topic'}
  className={css`
    margin: 3px ${spacing.xsmall} 3px auto;
    ${fonts.sizes(14, 1.1)};
  `}
   type="button"
   onClick={() => resourceSection && resourceSection.scrollIntoView()}>
>
{t('taxonomy.jumpToResources')}
</SomeTaxButton>
```

og i knappen da 
```
if (show){
  return null;
}
```

Noe sånt kanskje?